version/Cargo.toml

```diff
@@ -7,19 +7,18 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [features]
-default = ["scale", "json", "derive"]
-scale = ["parity-scale-codec"]
-json = ["serde_json"]
+default = ["std", "derive", "scale", "json"]
+std = ["iroha_macro/std", "parity-scale-codec/std", "thiserror"]
 derive = ["iroha_version_derive"]
+scale = ["parity-scale-codec/full"]
```

Comment:
This is why I have a question. Ideally, we should either align everything, or nothing. And in most `Cargo.toml` files in this PR the only things that are aligned (and then only within blocks), are the dependencies. Maybe we should roll this back to single space separation?

Reply:
Or maybe apply alignment consistently to all blocks, not just `dependencies`?

Reply:
I'll just revert this